core/sdk/src/api/device/device_request.rs

```diff
@@ -197,12 +198,15 @@ pub struct OnInternetConnectedRequest {
 
 #[derive(Debug, Serialize, Deserialize, Default, Clone)]
 pub struct LanguageProperty {
-    //#[serde(with = "language_code_serde")]
+    #[serde(with = "language_code_serde")]
     pub value: String,
 }
 
 #[derive(Debug, Serialize, Deserialize, Default, Clone)]
 pub struct TimezoneProperty {
+    // Original Regex in the Firebolt Timezone openrpc spec seems to be allowing
+    // even blank strings so the below test case is failing leaving it here
+    // so we can get a future resolution
     //#[serde(with = "timezone_serde")]
```

Comment:
So we are not doing strict timezone format validation here. Is the test cases still failing?

Reply:
Have used the Regex in the openrpc which is used in `serde_utils` it allows a blank timezone to be passed incidentally through the unit test. For now, let this code sit here while we fix the spec eventually we will uncomment it and have it working